controller/deis/settings.py

```diff
@@ -290,6 +290,9 @@
 # names which apps cannot reserve for routing
 DEIS_RESERVED_NAMES = ['deis']
 
+# the default expected status code for healthchecks
+DEIS_HEALTHCHECK_STATUS_CODE = '200'
```

Comment:
Ok. I think that's fine - if someone really wants this, they can use a custom component or figure out how to customize the setting in the general case.